riscv/insns/vmsgtu_vi.h

```diff
@@ -1,5 +1,6 @@
 // vsgtu.vi  vd, vd2, simm5
 VI_VI_ULOOP_CMP
 ({
-  res = vs2 > simm5;
+  uint64_t imm = (int8_t)(simm5 << 3) >> 3;
```

Comment:
The `v_simm5()` function already does this sign-extension to uint64_t; why do you need to do it again?